cedar-integration-tests/corpus_tests/9deac7d6cae0d5e05635469c985c01f9a22e0d1b.json

```diff
@@ -15,7 +15,7 @@
       "decision": "Deny",
       "reasons": [],
       "errors": [
-        "error occurred while evaluating policy `policy0`: record does not have the attribute: f"
+        "error occurred while evaluating policy `policy0`: record does not have the attribute: f. Available attrs: [\"\"]"
```

Comment:
CI is failing because this should be "attributes"

Reply:
😭